src/main/java/fi/helsinki/cs/tmc/cli/backend/Account.java

```diff
@@ -1,35 +1,53 @@
 package fi.helsinki.cs.tmc.cli.backend;
 
+import com.google.common.base.Optional;
+import fi.helsinki.cs.tmc.core.communication.oauth2.Oauth;
+import fi.helsinki.cs.tmc.core.domain.Course;
+import fi.helsinki.cs.tmc.core.domain.OauthCredentials;
+import fi.helsinki.cs.tmc.core.domain.Organization;
+
 /**
  * This object stores all login info.
  */
 public class Account {
 
-    static Account NULL_ACCOUNT = new Account(null, null, null);
+    static Account NULL_ACCOUNT = new Account(null, null, null, null);
```

Comment:
Null null null null null null null null null null null... :camel:

Reply:
The Null Account is a part of the original software. It is used when the method `setAccount(Account a)` of `Settings` is called with `a` being null, so that the getters wouldn't throw the `NullPointerException`. `setAccount` was added by the original team, and I did not find any use of it where `account` would seem to be null, so maybe deleting it would indeed be a good idea.